src/Uno.UWP/Media/Core/MediaSource.cs

```diff
@@ -1,18 +1,31 @@
 using System;
 using Windows.Media.Playback;
+using Windows.Storage.Streams;
 
 namespace Windows.Media.Core
 {
 	public partial class MediaSource : IDisposable, IMediaPlaybackSource
 	{
 		public Uri Uri { get; private set; }
 
+		public IRandomAccessStream Stream { get; private set; }
```

Comment:
This adds an API that is not present on Windows. I think this should be internal.